### ISSUE TITLE
Consider expanding labels

### ISSUE THREAD

**Issue by korikuzma** [biocommons/python-package#40]
**Is your feature request related to a problem? Please describe.**
This is not related to a problem, but more of a suggestion if others find this useful.

**Describe the solution you'd like**
I would like to propose to add the following labels: [github_labels.md](https://github.com/biocommons/biocommons.example/files/14017219/github_labels.md). 

<img width="611" alt="Screenshot 2024-01-22 at 8 40 31 PM" src="https://github.com/biocommons/biocommons.example/assets/42851808/4ef0f451-2b1f-4fc0-8d9b-ef13b68dfdc5">

The color of the labels does not matter to me, these are just the colors Wagner lab uses. 

I really like the priority labels, especially for PRs because it gives me an estimate of when I should make a review. It's also nice for helping decide which issues to tackle first.

The others would help contributors separate out the kinds of issues they like to tackle. It's also nice for conventional commits (which I would like to propose in the next developer meeting).

**Describe alternatives you've considered**
None

**Additional context**
None

**Comment by reece** [comment]
Agreed. Please see https://github.com/biocommons/infra/issues/2 for some links to other approaches. I particularly like https://seantrane.com/posts/logical-colorful-github-labels-18230/.
